### PR TITLE
feat: add use case of LLM on food review running on colab

### PR DESCRIPTION
**Pull request by jiashenC**

### CODE REVIEW

File: tutorials/14-food-review-tone-analysis-and-response.ipynb [hunk 1]
```diff
@@ -0,0 +1,1245 @@
+{
```
Comment (gaurav274) [inline tutorials/14-food-review-tone-analysis-and-response.ipynb:1]: <p>Can we use insert instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/georgia-tech-db/evadb/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (gaurav274) [inline tutorials/14-food-review-tone-analysis-and-response.ipynb:1]: <p>Can we load ChatGPT UDF by default?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/georgia-tech-db/evadb/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (gaurav274) [inline tutorials/14-food-review-tone-analysis-and-response.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #13.    for i in range(len(response_df)):
</pre><p>Why not just print the df?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/georgia-tech-db/evadb/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (jiashenC) [reply tutorials/14-food-review-tone-analysis-and-response.ipynb:1]: The response is a little long so I want to print out more. df print cuts out too much response.

Comment (jiashenC) [reply tutorials/14-food-review-tone-analysis-and-response.ipynb:1]: It turns out to be already part of bootstrap. I will delete it.

Comment (xzdandy) [reply tutorials/14-food-review-tone-analysis-and-response.ipynb:1]: Please check https://colab.research.google.com/notebooks/data_table.ipynb see if it helps

Comment (xzdandy) [inline tutorials/14-food-review-tone-analysis-and-response.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    %pip uninstall -y evadb
</pre><p>Can we clean this up? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/georgia-tech-db/evadb/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (jiashenC) [reply tutorials/14-food-review-tone-analysis-and-response.ipynb:1]: You meant just remove this?

Comment (xzdandy) [reply tutorials/14-food-review-tone-analysis-and-response.ipynb:1]: Yeah. Why we need to uninstall EvaDB first?

**Comment by jiashenC** [comment]
How do I add a colab link in the notebook? Is it just HTML?

**Review by gaurav274 — APPROVED** [review]

**Pull request merged**